x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -35,16 +36,15 @@ const initialData = {
 
 let hasDisplayedToast = false;
 
-function useServicesFetcher() {
+function useServicesFetcher({
+  environment,
+  kuery,
+}: {
+  environment: string;
+  kuery: string;
+}) {
   const {
-    urlParams: {
-      environment,
-      kuery,
-      start,
-      end,
-      comparisonEnabled,
-      comparisonType,
-    },
+    urlParams: { start, end, comparisonEnabled, comparisonType },
   } = useUrlParams();
```

Comment:
Is there any logic to which params are passed as function args, and which are retrieved from `useUrlParams`?

Reply:
I don't think I quite get what you mean, can you elaborate?